spec/models/service_spec.rb

```diff
@@ -2,9 +2,17 @@
 
 RSpec.describe Service, type: :model do
   context 'adding components to a service' do
+    let(:sp_component) { create(:sp_component) }
+    let(:msa_component) { create(:msa_component) }
+    let(:service) { create(:service, sp_component: sp_component) }
+
     it 'should create a new service correctly' do
-      service = create(:service, component: create(:sp_component))
+      expect(service).to be_valid
+      expect(service).to be_persisted
+    end
 
+    it 'should be able to have an sp and msa component' do
```

Comment:
May be this should be re-worded as there is no sp component added or this test should have failed if the intent was that both sp and msa component must on the service

Reply:
@jkunle `sp_component` is created with the `service` on line 7

Reply:
: